### DISCUSSION TITLE
Cannot color languages

### DISCUSSION THREAD

**Discussion by avi12** [community/community#47239] [Codespaces] [answered]
### Select Topic Area

Bug

### Body

Suppose I find a repo that contains a [Svelte file](https://github.com/TheComputerM/svelte-materialify/blob/master/packages/svelte-materialify/src/components/Slider/Slider.svelte); GitHub would happily color it
However, if I hop into VSCode Online (pressing <kbd>.</kbd>), it will fail to color it due to the [Svelte plugin](https://marketplace.visualstudio.com/items?itemName=svelte.svelte-vscode) being unavailable

**Comment by KartikM123** [comment, marked as answer]
Hi @avi12 !

Unfortunately, the team owning `Svelte for VSCode` needs to enable the extension for vscode web. As a workaround, if you are using a codespace, you can open your codespace from vscode desktop using the[ Codespaces extension](https://marketplace.visualstudio.com/items?itemName=GitHub.codespaces). There, the Svelte for VSCode extension should work!

**Reply by avi12** [reply]
Thank you for your information, however, I would like to use VSCode Online so that I don't have to clone anything to my machine and still inspect the code as if I'm using an IDE
